zippicks-master-critic-v2/includes/class-importer.php

```
@@ -0,0 +1,331 @@
+<?php
+/**
+ * JSON Importer for Master Critic
+ *
+ * @package ZipPicks_Master_Critic
+ */
+
+class ZipPicks_Master_Critic_Importer {
+    
+    private $errors = [];
+    private $imported_count = 0;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$imported_count'. (Unused Code Rules)

(UnusedPrivateField)